catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/FileSystemMapStoreFactory.java

```diff
@@ -31,6 +31,6 @@
   @Override
   public MapLoader<String, Object> newMapStore(String mapName, Properties properties) {
     LOGGER.trace("INSIDE: newMapStore()");
-    return new FileSystemPersistenceProvider(mapName);
+    return new FileSystemPersistenceProvider(mapName, properties);
```

Comment:
I'm thinking of changing this to just pass the directory into the provider instead of the properties object.